lib/dogapi/facade.rb

```diff
@@ -16,7 +16,7 @@ class Client
     #       [[api_key1, application_key1], [api_key2, application_key2]]
     # Hash to be able to specify different endpoints:
     #       {'https://app.example.com' => [[api_key1, application_key1], [api_key2, application_key2]]}
-    def initialize(api_key, application_key=nil, host=nil, device=nil, silent=true, timeout=nil, extra_endpoints=nil)
+    def initialize(api_key, application_key=nil, host=nil, device=nil, silent=true, timeout=nil)
```

Comment:
The comments above should be updated to remove any mention of `extra_endpoints`.